plugins/inventory/cobbler.py

```diff
@@ -177,6 +173,17 @@ def _add_safe_group_name(self, group, child=None):
             self.inventory.add_child(group_name, child)
         return group_name
 
+    def _exclude_profile(self, profile):
+        if len(self.include_profiles) > 0:
+            if profile in self.include_profiles:
+                return False
+            else:
+                return True
+        elif profile in self.exclude_profiles:
+            return True
+        else:
+            return False
```

Comment:
This can be simplified to:
```suggestion
            return profile not in self.include_profiles
        return profile in self.exclude_profiles
```